ci/push_images

```diff
@@ -12,12 +12,12 @@ badusage=64
 
 nprocs="${1:-1}"
 
-if [[ $TRAVIS_PULL_REQUEST != "false" ]] || [[ $TRAVIS_BRANCH != "develop" ]] ; then
+if [[ ${CURRENT_BRANCH} != "develop" ]] ; then
     echo "PR is not merged to develop, not pushing to DockerHub"
     exit 0;
 fi
 
-echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+echo "${DOCKERHUB_PASSWORD}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
```

Comment:
We store dockerhub parameters in Github actions with the names above

Reply:
Are they not secrets? How does this work?

Reply:
These are environment variables. I missed adding as environment variables added them